src/frethandfinger.ts

```diff
@@ -4,17 +4,24 @@
 // Class to draws string numbers into the notation.
 
 import { Builder } from './easyscore';
+import { FontInfo, FontStyle, FontWeight } from './font';
 import { Modifier } from './modifier';
 import { ModifierContextState } from './modifiercontext';
 import { StaveNote } from './stavenote';
-import { FontInfo } from './types/common';
 import { RuntimeError } from './util';
 
 export class FretHandFinger extends Modifier {
   static get CATEGORY(): string {
     return 'FretHandFinger';
   }
 
+  static TEXT_FONT: Required<FontInfo> = {
+    family: 'sans-serif' /* RONYEH: Font.SANS_SERIF */,
```

Comment:
3.0.9 hard coded font families in each individual element. Sometimes it would be 'sans-serif', other times it would be 'Arial'. Sometimes it was capitalized as 'Sans-serif' :-). I'd prefer to use a constant here. But for now I left it as is, because using the new constant will introduce a visual diff.

I'll submit a different PR soon after beta to resolve all of these TODOs.